gitlabform/processors/project/branches_processor.py

```diff
@@ -136,3 +133,7 @@ def transform_branch_config(self, branch_config: dict):
                             item["group_id"] = self.gl.get_group_id(item.pop("group"))
 
         return branch_config
+
+    @staticmethod
+    def check_for_wildcard(branch):
```

Comment:
Minor comment. The method name seems (to me) a little odd. It doesn't indicate what should happen in this method or if it will return something. Based on the method logic, since it may return a match or `None` maybe it can be something like `get_wildcard_branch_name`. Don't think that make sense either. Could call it `is_branch_name_wildcard` or something better and and in the logic, return boolean only.

Reply:
Sounds good. Thanks for the update.